index.html

```diff
@@ -0,0 +1,150 @@
+                <ul class="navList" id="navLi">
+                    <li class="closeBtn">&times</li>
+                    <li><a href="./about.html">About</a></li>
+                    <li><a href="#mainProgram">Program</a></li>
+                    <li><a href="#join">Join</a></li>
+                    <li><a href="#desktopSpeakers">Speakers</a></li>
+                    <li><a href="./about.html">Partners</a></li>
+                </ul>
+        </nav>
+        <section class="landingPage">
+                <div class="heading">
+                    <p>"Adorable Forex Trading"</p>
+                    <h1 class="image-Letter">FIRST YOU LEARN, THEN YOU TAKE OUT THE 'L'</h1>
+                </div>
+            
+                <div class="headingPara1">
+                    <P>Trading effectively is about assessing probabilities, 
+                        not certainties.” ― Yvan Byeajee</P>
+                </div>
+
+                <div class="headingPara2">
+                    <h4>8AM ~ 5PM MON ~ FRI</h4>
+                    <p>@ N0.31 Shopping Plaza, Old G.R.A, PH</p>
+                </div>
+        </section>
+       
+    </section>
+
+    <section>
+        <div class="mainProgram" id="mainProgram">
+            <p class="mainProgramP">Main Program</p>
+            <hr>
+            <div class="mainProDiv">
+                <div>
+                    <img src="images/account-Img.png" alt="">
+                    <h4>Investment</h4>
+                    <p>Get 15% of your investment; minimum of $500</p>
+                </div>
+                <div>
+                    <img src="images/learning-Img.png" alt="">
+                    <h4>Lecture</h4>
+                    <p>Listen to the tutors from various countries about trading</p>
+                </div>
+                <div>
+                    <img src="images/investment-Img.jpeg" alt="">
+                    <h4>Live Trading</h4>
+                    <p>Enjoy free live trading signals</p>
+                </div>
+                <div>
+                    <img src="images/account-Img.png" alt="">
+                    <h4>Account Management</h4>
+                    <p>Listen to the tutors from various countries about trading</p>
+                </div>
+                <div>
+                    <img src="images/training-Icon.png" alt="">
+                    <h4>Training</h4>
+                    <p>Enrolled students will be trained on how to milk the financial market</p>
+                </div>
+            </div>
+            <p class="join" id="join">Join Adorable Forex Trading</p>
+        </div>
+    </section>
+
+
+     <!-- Speakers -->    
+    <div class="fSpeakers">
+        <h3>Featured Speakers</h3>
+        <hr class="speakershr">
+    </div>
+    
+    <section class="desktopSpeakers" id="desktopSpeakers">     
+    </section> 
+    
+    <footer class="footer">   
+        <div class="footerPartner" id="footerPartner">
+            <p>Partner</p>
```

Comment:
- [ ] Your list of partners doesn't display as in the template. Kindly fix this please. You are expected to stick to design guidelines as much as you can.
### Your design
![image](https://user-images.githubusercontent.com/94551484/218193412-e85a20ee-53fc-4973-aa00-f19f988e3258.png)

### design guideline
![image](https://user-images.githubusercontent.com/94551484/218193324-3b7e3660-d182-47df-933c-048cff988e60.png)